api/pyproject.toml

```diff
@@ -142,7 +142,7 @@ environs = "~9.2.0"
 django-lifecycle = "~1.0.0"
 drf-writable-nested = "~0.6.2"
 django-filter = "~2.4.0"
-flagsmith-flag-engine = "^5.1.1"
+flagsmith-flag-engine = { git = "https://github.com/Flagsmith/flagsmith-engine", branch = "feat/add-dashboard-alias"}
```

Comment:
Is this change meant for the merge with main or is it just a WIP change?

Reply:
Sorry, no, good catch. I need [this PR](https://github.com/Flagsmith/flagsmith-engine/pull/210) merged and released before we merge this one.

Reply:
This is done now.